src/azul/health.py

```diff
@@ -108,6 +110,32 @@ def queues(self):
                 }
         return response
 
+    @health_property
+    def failures(self):
```

Comment:
I've arrived at the conclusion that it doesn't make sense to expose the failures as part of a standard /health response and then only return the first 100. Fetching 100 items is both too much for a normal health response and too little for a client that is interested in looking at all errors. This should be exposed as a separate `/health/failures` (not `/health/{keys}`) and a ticket should be created to add paging to that endpoint.

The mocks that were added to the tests of `/health` should be remove again.